patches/minecraft/net/minecraft/world/gen/layer/GenLayer.java.patch

```diff
@@ -39,9 +48,9 @@
 +        return j;
 +    }
 +
-+    public static int getModdedBiomeSize(WorldType worldType, int original)
++    public static int getModdedBiomeSize(@javax.annotation.Nullable net.minecraft.world.World world, WorldType worldType, int original)
```

Comment:
Public methods like this can't be changed directly, they need to be deprecated and replaced with new methods.